include/grpc/impl/codegen/compression_types.h

```diff
@@ -101,15 +100,15 @@ typedef struct grpc_compression_options {
    * precedence over \a default_algorithm.
    * TODO(dgq): currently only available for server channels. */
   struct {
-    bool is_set;
+    int is_set;
```

Comment:
Did this C89 violation ever make it into a release branch? If so, should it be backported?